### PR TITLE
docs(blind): update ux styleguide

### PR DESCRIPTION
**Pull request by linstedt**
Please check the type of change your PR introduces:

- [ ] Bug fix
- [ ] Feature
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/documentation/docs/controls/_blind_styleguide.md [hunk 1]
```diff
@@ -10,76 +10,58 @@ import SourceAngularHtml from './../auto-generated/previews/angular/blind.html.m
...
+Multiple blind variants are available: 
+- **Filled**: Default variant
+- **Outline**: Variant for lower visual emphasis
+- **Primary**: Variant for high visual emphasis
+- **State-related variants**: Alarm, critical, warning, success, info, neutral
 
-### When not to use
 
-Consider the context in which the blinds will be used. If the content is central to the user's task, using a blind might reduce the content's visibility and accessibility.
+![Blind variants](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=929%3A47485&mode=design&t=9faEnH99BaAxqCGM-1)
```
Comment (danielleroux) [inline packages/documentation/docs/controls/_blind_styleguide.md:36]: Illustration on figma shows no differnet variants:
image

File: packages/documentation/docs/controls/_blind_styleguide.md [hunk 2]
```diff
@@ -10,76 +10,58 @@ import SourceAngularHtml from './../auto-generated/previews/angular/blind.html.m
...
-- [States](#states)
-- [Dos and don't](#dos-and-donts)
+1. Header section
+2. Content section
 
-## Usage
+## Types
 
-Blinds are often used to display a large amount of content in a compact space, or to present information in an organized and hierarchical way.​ Blinds can reduce cognitive load of the user be removing clutter and less important information from an interface.
+Multiple blinds can be placed below each other to create an accordion. The recommended distance between the blinds is `0.5rem`. Blinds within an accordion can either close and open independently or dependently depending on the design requirements.
```
Comment (nuke-ellington) [inline packages/documentation/docs/controls/_blind_styleguide.md:24]:
```suggestion
Multiple blinds can be placed below each other to create an accordion. The recommended distance between the blinds is `0.5rem`. The respective design requirements dictate whether it is allowed have multiple open blinds or if there can only be one blind open at a time.
```

File: packages/documentation/docs/controls/_blind_styleguide.md (hunk 1 above)
Comment (linstedt) [reply packages/documentation/docs/controls/_blind_styleguide.md:36]: Good catch, it's fixed now

**Review by nuke-ellington — DISMISSED** [review]

File: packages/documentation/docs/controls/_blind_styleguide.md [hunk 3]
```diff
@@ -10,76 +10,58 @@ import SourceAngularHtml from './../auto-generated/previews/angular/blind.html.m
 
 import Playground from '@site/src/components/Playground'
 
-Blinds are UI controls that allow the users to hide or reveal content by clicking on a control element.
+Blinds are UI controls that allow the users to hide or reveal content by clicking on a control element. Blinds can display a large amount of content in a compact space or present information in an organized and hierarchical way. Blinds reduce the user's cognitive load by removing clutter and less important information from an interface. We typically don't use blinds if the content is central to the user's task due to its reduced visibility and accessibility.
```
[the remark this answers is not in the document]
Comment (linstedt) [reply packages/documentation/docs/controls/_blind_styleguide.md:13]: Propositions are not in line with language check done together with Jennie.

File: packages/documentation/docs/controls/_blind_styleguide.md (hunk 2 above)
Comment (linstedt) [reply packages/documentation/docs/controls/_blind_styleguide.md]: I tried to fix the description to make it more clear and straightforward. Unfortunately, I couldn't reach Jennie for a review.

**Review by danielleroux — APPROVED** [review]

**Pull request merged**